data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanBucketOption.java

```diff
@@ -26,11 +26,11 @@ public class S3ScanBucketOption {
     @JsonProperty("start_time")
     private LocalDateTime startTime;
 
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonDeserialize(using = CustomLocalDateTimeDeserializer.class)
     @JsonProperty("end_time")
     private LocalDateTime endTime;
 
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonDeserialize(using = DurationDeserializer.class)
```

Comment:
This actually isn't needed because it is handled by the data-prepper-core deserializer